frontend/apps/reader/modules/readerfont.lua

```diff
@@ -331,7 +331,7 @@ function ReaderFont:makeDefault(face, is_monospace, touchmenu_instance)
             -- to be set as a fallback font, and allow it to be set as the
             -- default monospace font.
             UIManager:show(MultiConfirmBox:new{
-                text = T(_("Would you like %1 to be used as the default font (★), or the monospace font (\u{1F13C})?"), face),
+                text = T(_("Would you like %1 to be used as the default font (★), or the monospace font (\xF0\x9F\x84\xBC)?"), face), -- U+1F13C
```

Comment:
I was unsure if the codepoints were standard or not (e.g., pulled from our custom NerdFont ranges), given that they were escaped (i.e., I assumed the escaping was for that reason ;)).

Reply:
> I assumed the escaping was for a/that reason ;)

`A` reason :)  `my` reason: I still use latin1 terminals/environment, so utf8 reads like crap to me, and even if some symbols like these ones are readable, others like some punctuations or spaces are not, and using `\u{2022}` for anything non-ascii feels just clearer.

For these ones, I dunno, lookup the utf8 sequence (or the unicode codepoint) on Google and see if they are proper Unicode.

If not, they are from nerdfont.

> I don't understand that? Do you mean, that by any chance, some fonts have a different mapping for some symbols? 

No, but when some symbols are not in Unicode, we can use our nerdfont.ttf, which maps some glyphs into Unicode private ranges - that we then can use: the glyphs won't be found in any of our normal fonts, but will be in nerfond.ttf, which is in our fallback fonts list.

Reply:
We shouldn't have any of your ä or my french accents in the code base :)

> So (not offending) what the fuck is this argument for? 

That wasn't an argument, I was just explaining the situation :)

I'm not opposed to using ★ and similarly apprehendable characters in translated strings, and in the code.
But for other less obvious stuff, like non-ascii punctuations/space (ie. thin or hair spaces, that I think you had used before), I'd rather see them as `\u{xyzx}` than you and me not noticing them at all. Also, it it's a bunch (like I have in wikipedia.lua), I'd want to see as a comment -- `U+1234` so it's easy to get to it, see in which Unicode block it is to find and pick similar neighbours - rather than having to copy and paste ★ into Google (which doesn't always work as well as for this one).

> stay in the year U+00C4)1990 (or before, latin1 sound a really solid old school 

These were good years, and good old school :)

Reply:
> We shouldn't have any of your ä or my french accents in the code base :)

They do fit in English, although I'm not quite sure when we'd use a word like fiancé(e) or even déjà-vu. Perhaps we could sneak in a métadata somewere.

Reply:
FWIW, since I'm using ZSH + hexdump for the conversion, the only check I did was whether I got a tofu or not before hexdump ;).

Evidently, these are not covered by Consolas, so I assumed they weren't standard NF ;).

Regardless, even on "legacy" setups, I would think whatever happens to the native glyph would feels slightly less worse than raw byte escapes here ;).

Reply:
> the only check I did was whether I got a tofu or not before hexdump ;).

Turns out I'm just blind, Consolas does cover the `🄼` glyph, I just mistook it for a sand glass, so i just assumed it was from NF ;D